docker/Dockerfile.amd

```diff
@@ -147,13 +153,31 @@ ENV DEVITO_ARCH="aomp"
 ENV DEVITO_PLATFORM="amdgpuX"
 ENV DEVITO_LANGUAGE="openmp"
 
-
 ########################################################################
 # HIPCC for GPUs (HIP)
 ########################################################################
 # This will only trigger if arch is hip since the final stage depends on it
 FROM sdk-base as hip
 
+# Install OpenMPI
```

Comment:
why was OpenMPI expanded out ? there must be a logic, but I'm missing it

Reply:
ah, for CC=, CXX= etc...

Reply:
Yeah, decided to go concervative and force all the compiler for openmpi we can revisit